kubetest/e2e.go

```diff
@@ -37,6 +37,9 @@ func run(deploy deployer, o options) error {
 	os.Setenv("KUBE_CONFIG_FILE", "config-test.sh")
 	// force having batch/v2alpha1 always on for e2e tests
 	os.Setenv("KUBE_RUNTIME_CONFIG", "batch/v2alpha1=true")
+	// force having apps/v1beta2 always on for e2e tests
+	// TODO: remove it once we enable apps/v1beta2 by default
+	os.Setenv("KUBE_RUNTIME_CONFIG", "apps/v1beta2=true")
```

Comment:
test context is here: https://github.com/kubernetes/kubernetes/blob/master/test/e2e/framework/test_context.go

How is this environment variable used?

We are trying very hard to eliminate implicit dependencies on global variables. We should identify which programs kubetest calls need this config and explicitly send it to those programs.

Also, we should be able to modify this value without shipping a new kubekins image. Will you convert this into a kubetest flag? https://github.com/kubernetes/test-infra/blob/master/kubetest/main.go#L92 

Is this setting really the right one for every job? Including ones from say 1.5? If not then let's update `jobs/config.json` to add this `--new-flag=batch/v2alpha1=true,apps/v1beta2=true` to the relevant jobs